Agile Development/Burndown Chart/script.py

```diff
@@ -0,0 +1,132 @@
+import pprint
+import requests
+import jpholiday
+import datetime
+import matplotlib.dates as mdates
+import pandas as pd
+import matplotlib.pyplot as plt
+import urllib.request
+import urllib.parse
+import json
+import datetime
+from pytz import timezone
+point_dict = {}
+# ---- #
+# init #
+# ---- #
+total_points = 0
+done = 0
+undone = 0
+BASIC = 'Basic XXXXXXXXXXXXXXX=='
+SPRINT = "Sprint1"
+# ---------- #
+# Get Sprint #
+# ---------- #
+params = {
+    'sysparm_query': 'short_description=' + SPRINT
+}
+param = urllib.parse.urlencode(params)
+url = "https://<InstanceName>.service-now.com/api/now/table/rm_sprint?" + param
+req = urllib.request.Request(url)
+req.add_header("authorization", BASIC)
+with urllib.request.urlopen(req) as res:
+    r = res.read().decode("utf-8")
+obj = json.loads(r)
+# Get the start and end dates of a Sprint
+start_date = obj['result'][0]['start_date']
+start_date = (datetime.datetime.strptime(start_date, '%Y-%m-%d %H:%M:%S') + datetime.timedelta(hours=9)).date()
+print(start_date)
+end_date = obj['result'][0]['end_date']
+end_date = (datetime.datetime.strptime(end_date, '%Y-%m-%d %H:%M:%S') + datetime.timedelta(hours=9)).date()
+# Initializing the points array
+while start_date <= end_date:
+    point_dict[str(start_date)] = 0
+    start_date = start_date + datetime.timedelta(days=1)
+# ----------- #
+# Storyを取得 #
+# ----------- #
+params = {
+    'sysparm_query': 'sprint.short_descriptionLIKE' + SPRINT
+}
+p = urllib.parse.urlencode(params)
+url = "https://<InstanceName>.service-now.com/api/now/table/rm_story?" + p
```

Comment:
Instead of variables in the script, can you switch to using [argparse](https://docs.python.org/3/library/argparse.html) and taking these as inputs via command line so a user doesn't need to open the script and modify values every time?